test/compile.js

```diff
@@ -45,10 +45,22 @@ test('compile - without tr.compile', function () {
   var tr = createTransformer({
     name: 'test',
     outputFormat: 'html',
-    render: function () {
+    compileClient: function () {
     }
   });
   assert.throws(function () {
     tr.compile('example input', {});
   }, /does not support compilation/);
 });
+
+test('compile - without tr.compile, but with tr.render => fn', function () {
+  var tr = createTransformer({
+    name: 'test',
+    outputFormat: 'html',
+    render: function (str, options, locals) {
+      assert(str === 'example input');
+      return locals.name;
+    }
+  });
+  assert.equal(tr.compile('example input').fn({name: 'hola'}).body, 'hola');
```

Comment:
Here, the `.fn` is supposed to return the body directly, not an object with a key called `body`.